controllers/AuthController.js

```diff
@@ -51,36 +49,32 @@ class AuthController {
         const payload = {
           message: 'This email has been taken'
         };
-        return statusResponse.conflict(res, payload);
-      }
-      try {
-        const emailVerification = 'false';
-        const userData = await users.create({
-          email,
-          username,
-          emailVerification,
-          password: hashPassword
-        });
-        await roles.create({
-          role,
-          userId: userData.id
-        });
-        const token = jwt.sign({ email, username }, process.env.TOKEN_SECRET, {
-          expiresIn: 86400
-        });
-        delete userData.dataValues.password;
-        const payload = {
-          message: 'user created succesfully',
-          userData,
-          token,
-          emailToken
-        };
-        return statusResponse.created(res, payload);
-      } catch (error) {
-        return statusResponse.internalServerError(res);
+        return StatusResponse.conflict(res, payload);
       }
+      const emailVerification = 'false';
```

Comment:
Good work @edogbosunny in making this code refactor, however I think the emailVerification should be a boolean data type with a default value set to fault in the database model. Kindly implement this and I think all should be fine here. Cheers.